content/applications/sales/subscriptions/products.rst

```diff
@@ -1,61 +1,106 @@
-==================
-Recurring products
-==================
+=====================
+Subscription products
+=====================
 
-When creating a subscription, at least one product on the order lines must be configured as
-:ref:`recurring <subscriptions/product/recurring>`.
+By integrating closely with the Odoo *Sales* app, the *Subscriptions* app enables users to sell
+subscription products alongside regular sales products. While regular products are sold on a
+one-time basis, subscription products are sold on a renewing basis, generating recurring revenue.
 
-If a single product is sold using more than one recurrence period, configure the product's
-:ref:`time-based pricing <subscriptions/product/time-based-pricing>` to automatically adapt the unit
-price on the subscription based on the recurrence set.
+In Odoo, subscription products are also called *recurring* products.
 
-.. _subscriptions/product/recurring:
+Configure recurrence periods
+============================
 
-Configuration
-=============
+To get started with subscriptions, first make sure that the *recurrence periods* are configured as
+needed.
 
-To create a recurring product, go to :menuselection:`Subscriptions --> Subscriptions --> Products`
-and click :guilabel:`New`. Choose a :guilabel:`Product Name` and leave :guilabel:`Recurring` and
-:guilabel:`Can be Sold` enabled.
+Recurrence periods are the time periods in which subscriptions renew. They designate how often the
+customer pays for (and receives) subscription products.
 
-.. _subscriptions/product/time-based-pricing:
+To configure recurrence periods, go to :menuselection:`Subscriptions --> Configuration -->
+Recurrence Periods`.
+
+The *Subscriptions* app comes with some basic recurrence periods already configured: Daily, Monthly,
+Quarterly, Weekly, Yearly, 3 Years, and 5 Years. These can be edited as needed, and any number of
+new recurrence periods can be added.
```

Comment:
this could also be a note since it seems like supplemental information to the active configuration steps